lua/nvim-surround/init.lua

```diff
@@ -53,12 +56,9 @@ M.insert_surround = function(line_mode)
     -- Indent the cursor to the correct level, if added line-wise
     if line_mode then
         local lnum = buffer.get_curpos()[1]
-        if vim.bo.expandtab then
-            buffer.insert_lines({ lnum, 1 }, { (" "):rep(vim.fn.indent(lnum + 1) + vim.bo.shiftwidth) })
-        else
-            local num_tabs = vim.fn.indent(lnum + 1) / vim.bo.shiftwidth
-            buffer.insert_lines({ lnum, 1 }, { ("	"):rep(num_tabs + 1) })
-        end
+
+        buffer.insert_lines({ lnum, 1 }, { "" })
```

Comment:
This line is actually a NOOP, since the `insert_lines` function adds lines to the buffer *between* two characters. What I had before was adding whitespace, so it actually did do something.

Reply:
I could've sworn that line was necessary at some point, but it must've been something else as everything seems to work fine when I remove it.

I was working under the assumption that `insert_lines()` inserted *lines* in the Vim sense with implicit line breaks on either side of the insert point.

Reply:
Yeah that might be a bit of a bad name on my part; I meant it to insert exactly between two characters and "push everything else aside", similar to how `p` works.